ui/packages/catalog-ui-search/src/main/webapp/js/OpenLayersGeometryUtils.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global require, module*/
+/*jshint esversion: 6, bitwise: false*/
+const ol = require('openlayers');
+const properties = require('properties');
+const Common = require('./Common');
+
+module.exports = {
+    getCoordinatesFromGeometry: geometry => {
+        const type = geometry.getType();
+        switch (type) {
+            case 'LineString':
+                return geometry.getCoordinates();
+            case 'Polygon':
+                return geometry.getCoordinates()[0];
+            case 'Circle':
+                return [geometry.getCenter()];
+            default:
+                return [];
+        }
+    },
+    setCoordinatesForGeometry: (geometry, coordinates) => {
+        const type = geometry.getType();
+        switch (type) {
+            case 'LineString':
+                geometry.setCoordinates(coordinates);
+                break;
+            case 'Polygon':
+                geometry.setCoordinates([coordinates]);
+                break;
+            case 'Circle':
+                geometry.setCenter(coordinates[0]);
+                break;
+            default:
+                break;
+        }
+    },
+    mapCoordinateToLonLat: point => ol.proj.transform(point, properties.projection, 'EPSG:4326'),
```

Comment:
❓ Why do we need to transform the projection of the geo into 4326?  If the configured layer projection is something other than 4326 wouldn't this cause an issue of the resulting geolocation of the geo on the map?

Reply:
`grep 'EPSG:4326' -Rl . --exclude-dir node_modules --exclude-dir target --include=\*.{js,java}`
Returns 36 files that contain that constant. If it is variable then we need to change most if not all of them.

Reply:
https://github.com/codice/ddf/blob/master/catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.config.xml#L52